skia/dependencies/get_skia.sh

```diff
@@ -32,6 +32,11 @@ getSkia () {
     cd .. 
 }
 
-getSkia https://github.com/rive-app/skia rive skia_rive_optimized
-
-getSkia https://github.com/google/skia chrome/m99 skia
\ No newline at end of file
+if [ $# -eq 0 ]; then
+    # No arguments supplied; checkout the default skia repos.
+    getSkia https://github.com/rive-app/skia rive skia_rive_optimized
+    getSkia https://github.com/google/skia chrome/m99 skia
```

Comment:
Looks like the new Skia milestone is v102. Should we upgrade?

Reply:
That seems like a good idea, for a different PR?